FordLabs/PeopleMover
# 132 read only redirect

## Issue
Resolves #132 
Authenticate View Only Users to Space - not dashboard

## What was done
- [x] Introduce SessionStorage variable to capture the space uuid the user may be going to when getting redirected through ADFS
- [x] Add matcher route for H2 DB on backend
- [x] Fix lint warning

## How to test
1. Log out of test environment
2. Go to read only space in that environment
3. Go through ADFS authentication process
4. Verify redirected to the read only space

### FOR DEVS
- Chose Session storage because it has a shorter lifespan than local storage
- Could have done this through ADFS Domain Whitelist, but that would required wildcard matching, requiring a lot of approvals & time
- Thought about doing this in Routes, but there wasn't an easy way to test it there. OAuthRedirect was easier to test and was only used in one place to begin with.
